app/src/main/res/values/strings.xml

```diff
@@ -46,14 +46,14 @@
     <string name="sensor_accuracy_low">Low accuracy</string>
     <string name="sensor_accuracy_medium">Average accuracy</string>
     <string name="sensor_accuracy_high">Maximum accuracy</string>
-    <string name="sensor_accuracy_image_description">Indicates graphically the sensors accuracy</string>
+    <string name="sensor_accuracy_image_description">Indicates graphically the sensor\'s accuracy</string>
```

Comment:
By looking at the rest of the document I saw that you were always referring to a single sensor rather than multiple so I adapted it in this sentence too thinking that it was what you intended.  
Te `'s` here isn't meant as an elision of "is" but as the possessive (Saxon genitive): "sensor's accuracy" -> "the accuracy of the sensor"

Reply:
Anyway I also agree that dropping the `s` entirely would work as well, the meaning remains the same